infrastructure/modules/aws/eks/patch-coredns.tf

```diff
@@ -3,7 +3,7 @@ resource "null_resource" "patch_coredns" {
   provisioner "local-exec" {
     command = "kubectl -n kube-system patch deployment coredns --patch \"${yamlencode(local.patch_coredns_spec)}\""
     environment = {
-      KUBECONFIG = module.eks.kubeconfig_filename
```

Comment:
I think I prefer the way it was done before, as it forces the dependency on the module.

Reply:
done